### DISCUSSION TITLE
How to Limit Display of Selected Tags and Show Count Instead

### DISCUSSION THREAD

**Discussion by alahanoun** [vueform/vueform#344] [Questions]
I’m currently using vueform’s TagsElement, and I would like to customize how the selected tags are displayed.

I want to limit the number of selected tags displayed to a certain maximum (e.g., 5 tags). After reaching that limit, I’d like to replace the tags with the number of selected options, similar to the MultiSelectElement: 

![image](https://github.com/user-attachments/assets/b8370cac-8e98-470c-a766-c3fb0a9221e4)

Is there a way to achieve this in vueform?

Thank you for your help!

**Comment by adamberecz** [comment]
Unfortunately there's no native way to achieve that. You can limit the tags but there's no slot for placing anything after them.
